BUILD

```diff
@@ -66,15 +52,9 @@ cc_library(
         "include/cctz/zone_info_source.h",
     ],
     includes = ["include"],
-    linkopts = select({
-        "//:osx": [
-            "-framework Foundation",
-        ],
-        "//:ios": [
-            "-framework Foundation",
-        ],
-        "//conditions:default": [],
-    }),
+    # Note that OS X and iOS both have an implicit dependency on Foundation but
+    # it is no longer explicitly added as a linkopt as bazel adds it
+    # automatically. See https://github.com/abseil/abseil-cpp/issues/326 for details.
```

Comment:
the way bazel adds it doesn't cause problems for android builds on macos.
Given this fixes known bugs for multiple downstream customers and the only down-side is an implicit rather than explicit dependency, would it be possible to file a tracking issue for you folks to investigate what bazel does and add back a (working) explicit dependency?  I don't know enough about bazel internals to know what they're doing and how it works but I know from again the multiple reports of breakage with explicit includes that there is a real problem I would like to fix, without the perfect solution stopping a known improvement.